Scripts/jazzy.sh

```diff
@@ -0,0 +1,13 @@
+ver=`/usr/libexec/PlistBuddy -c "Print CFBundleShortVersionString" ParseSwift-iOS/Info.plist`
+jazzy \
+  --clean \
+  --author "Parse Community" \
+  --author_url http://parseplatform.org \
+  --github_url https://github.com/parse-community/Parse-Swift \
+  --root-url http://parseplatform.org/Parse-Swift/api/ \
```

Comment:
@TomWFox I just "made this up" based off of the other projects, not sure if you want this to be the link

Reply:
I'm happy with that. The obj-c SDK currently published the jazzy docs on release (although it's broken rn!) - I'm thinking we could set this SDK to do it every commit - for now at least - what do you think?

Reply:
Sounds good to me!